### ISSUE TITLE
Docs that need work / repairs in sandbox section.

### ISSUE THREAD

**Issue by maximuskowalski** [saltyorg/docs#129]
Have created links and barebones template docs for following apps that were missing from the sandbox index.yml

They all need completing and filling out - URLS to docs and project page and they will need adding to the `mkdocs.yml` file when completed / repaired. If you fix one of these maybe mention it here or copy below list and remove the doc you have worked on when you paste it in a comment for the next person to look at - I will try to edit this post to reflect work done.

```yaml
sandbox/apps/a_train.md
sandbox/apps/changedetection.md
sandbox/apps/cherry.md
sandbox/apps/freshrss.md
sandbox/apps/gotenberg.md
sandbox/apps/homebox.md
sandbox/apps/mcrouter.md
sandbox/apps/plex_auto_languages.md
sandbox/apps/plex_autoscan.md
sandbox/apps/plex_patrol.md
sandbox/apps/plextraktsync.md
sandbox/apps/plex_utills.md
sandbox/apps/pyload.md
sandbox/apps/python-plexlibrary.md
sandbox/apps/qbittorrentvpn.md
sandbox/apps/rdtclient.md
sandbox/apps/rocketchat.md
sandbox/apps/sarotate.md
sandbox/apps/teamspeak.md
sandbox/apps/tika.md
sandbox/apps/transmissionvpn.md
sandbox/apps/whisparr.md
sandbox/apps/xbackbone.md
```

**Comment by RaneyDazed** [comment]
well well well. Looks like a job for a CHAIR. fr fr

**Comment by maximuskowalski** [comment]
Ah - perfect. A checklist makes it much easier. Thanks Owine.

**Comment by RaneyDazed** [comment]
Yeah, great idea owine! 🫠

**Comment by maximuskowalski** [comment]
phpmyadmin also missing

**Comment by RaneyDazed** [comment]
#188 clears sarotate, and #196 will clear whisparr from the list.

**Comment by nwithan8** [comment]
#221 adds tauticord documentation

**Comment by RaneyDazed** [comment]
should be able to clear changedetection, freshrss, homeassistant, Krusader, mcrouter, mqtt, and node-red after #273. Some were already done by now.

**Comment by saltydk** [comment]
status on this @maximuskowalski @RaneyDazed ?

**Comment by RaneyDazed** [comment]
> status on this @maximuskowalski @RaneyDazed ?

@saltydk ill take another look over it tonight, I won't be able to do anything with it tomorrow though.

**Comment by keldian** [comment]
https://github.com/saltyorg/docs/pull/338/commits/76f17f48886019ade128c097474305549bc08ad0 :white_check_mark: teamspeak
